src/libraries/Common/tests/System/Security/Cryptography/AlgorithmImplementations/RSA/EncryptDecrypt.cs

```diff
@@ -336,11 +337,19 @@ private void RsaCryptRoundtrip(RSAEncryptionPadding paddingMode, bool expectSucc
             Assert.Equal(TestData.HelloBytes, output);
         }
 
-        [Fact]
+        [ConditionalFact]
         [SkipOnTargetFramework(TargetFrameworkMonikers.NetFramework)]
-        [ActiveIssue("https://github.com/dotnet/runtime/issues/52199", TestPlatforms.iOS | TestPlatforms.tvOS | TestPlatforms.MacCatalyst)]
         public void RoundtripEmptyArray()
         {
+            if (PlatformDetection.IsiOS && !OperatingSystem.IsIOSVersionAtLeast(13, 6))
```

Comment:
nit: use the `OperatingSystem` API directly rather than going through the `PlatformDetection` helper:

```suggestion
            if (OperatingSystem.IsIOS() && !OperatingSystem.IsIOSVersionAtLeast(13, 6))
```

Reply:
Thanks, in other scenarios where we do use `PlatformDetection.Is*` why wouldn't we want to directly use `OperatingSystem` instead?

Reply:
If there's an equivalent API in `OperatingSystem` I'd always use that one, it's mostly historical reasons we're not doing it right now since `PlatformDetection` was added much earlier than `OperatingSystem`.